server.py

```diff
@@ -71,7 +82,9 @@ class Meta:
 
 
 db.connect()
-db.create_tables([User, Session, Submission, ContestProblems, Contest, Question])
+db.create_tables(
+    [User, Session, Submission, ContestProblems, Contest, Question, UploadedQuestion]
```

Comment:
why two question classes? Let's replace the old Question with your new one.

Reply:
Until the code was properly reviewed, I dint want to interfere with the already existing code